examples/next/README.md

```diff
@@ -0,0 +1,29 @@
+## Setting up
+In order to try out these examples, you need to set your public API key as the value of the NEXT_PUBLIC_FPJS_PUBLIC_API_KEY variable\
+in the `.local.env`, you can just copy `.env` file, located in this directory.
+
+### FingerprintJS public API key
+- Go to [FingerprintJS Dashboard](https://dashboard.fingerprintjs.com/)*
+- Open the _API keys_ page from the sidebar
+- Find your _Public_ API key
+
+*If you don't have a FingerprintJS Pro account, [sign up for free](https://dashboard.fingerprintjs.com/signup/)).
+
+### Installing dependencies
+1. Go to the root of the project, install dependencies and build it (`cd ../../ && yarn install && yarn build`)
```

Comment:
Wouldn't it better to create these scripts in the package.json ant then ask them to run just one or two commands?

Reply:
I suppose it is overengineering to create a separate script. Rewrote the text for better perception.